service/docs/source/security.rst

```diff
@@ -1,192 +1,167 @@
-or two inter-process shared memory regions to interact with the
-client: one region provides a user interface to read signals, and the
-other if requested provides a user interface to write
-controls. Requests to read, write, or quit are made through a FIFO
-channel connecting the client to the batch server, and a second FIFO
-is used to communicate to the client when requests have been
-completed. These FIFOs are opened in ``/run/geopm/batch-status-*``.
-
-
-Secure restart of the ``geopmd`` daemon process
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The GEOPM Service only supports one active writing user session at a time.
+This restriction supports the save/restore methodology explained in the
+previous section. The write lock enabling this mutual exclusion is tied
+to the Unix "process session leader" PID of the requesting process. If
+this session leader PID doesn't refer to an active process, the lock
+is associated with the requesting process PID. Otherwise, any PID within
+the requesting Unix process session can write via the GEOPM Service for
+the entire process session leader PID lifespan.
 
-All state that is required to maintain the security guarantees of the
-GEOPM Service are stored to disk in a secure way (see
-:ref:`security:File Usage/Configuration`
-for more details).  This information is read from disk when the
-``geopmd`` daemon process begins. This enables the service to cleanly
-recover from any premature ending of the daemon process. The restart
-of the daemon process is enabled by the systemd Linux service though a
-configuration option that the GEOPM Service provides. These options
-enable the GEOPM Service to be restarted immediately, and to attempt
-to restart twice in rapid succession if necessary. The first attempt
-may be used to provide a clean environment for the second restart.
+
+Providing a Low-Latency Interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The GEOPM Service offers a batch interface letting users configure
+a server process. The batch server process is forked by ``geopmd``,
+the primary GEOPM Service daemon process started by `systemd(1)
+<https://man7.org/linux/man-pages/man1/systemd.1.html>`__.  The user can
+configure the batch server with an array of requests via DBus. Access
+permissions are checked when initiating the server, which will only be
+forked if the client has authorization for all array requests.  The batch
+server establishes one or two inter-process shared memory regions to
+communicate with the client — one for user interface signal readings,
+and another (if requested) to provide the user interface control for
+writing.  The client makes requests to read, write, or quit through
+a FIFO channel, connecting them to the batch server. A second FIFO
+communicates to the client when requests are complete. These FIFOs are
+opened in ``/tmp``, which systemd sets up by default as a `tmpfs(5)
+<https://man7.org/linux/man-pages/man5/tmpfs.5.html>`__.
```

Comment:
these FIFOs are in /run/geopm.  I think Daniels correction to dev may have been clobbered.